<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>三菱PLC</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="BTN_OpenPLC">
        <property name="text">
         <string>打开PLC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout1">
        <item row="0" column="0">
         <widget class="QLabel" name="label1">
          <property name="text">
           <string>设备名称:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_DeviceName">
          <property name="text">
           <string>D1324</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="BTN_GetValue">
          <property name="text">
           <string>获取数值</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label2">
          <property name="text">
           <string>设备数值:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_DeviceValue">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="BTN_SetValue">
          <property name="text">
           <string>设置数值</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="BTN_GetCPUType">
        <property name="text">
         <string>获取CPU类型</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_ActProgType">
          <property name="text">
           <string>ActProgType</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioButton_ActUtlType">
          <property name="text">
           <string>ActUtlType</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="BTN_ClosePLC">
        <property name="text">
         <string>关闭PLC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="title">
      <string>西门子PLC</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout3">
        <item row="0" column="2">
         <widget class="QPushButton" name="BTN_ConnectPLC">
          <property name="text">
           <string>连接PLC</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label3">
          <property name="text">
           <string>PLC地址:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_IP">
          <property name="text">
           <string>192.168.1.120</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_DeviceValue_XiMenZi">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>设备地址:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="BTN_SetValue_XiMenZi">
        <property name="text">
         <string>设置数值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="BTN_GetValue_XiMenZi">
        <property name="text">
         <string>获取数值</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>设备数值:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_DeviceAdress_XiMenZi">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="BTN_ClosePLC_XiMenZi">
        <property name="text">
         <string>关闭PLC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
